### ISSUE TITLE
"show": "right" makes keyboard diagrams disappear

### ISSUE THREAD

**Issue by nomike** [ChordPro/chordpro#236]
## Describe the bug
When you add the setting
"pdf" -> "diagrams" -> "show" : "right"
no diagrams are sown. In fact, even if you set the value to "top", "bottom" or "below", they just disappear.

I'm also questioning the decision to show guitar chords in the right column by default and keyboard diagram on the bottom of the first page. Even as it's documented, why treat them differently?

### To Reproduce
Download the example and run the Makefile.

### Expected behavior
Diagrams for keyboard should be shown on the correct position depending on the config and should not disappear.

### Attachments
[piano.tar.gz](https://github.com/ChordPro/chordpro/files/9275879/piano.tar.gz)

## System information
 - OS: Ubuntu 22.04 LTS
 - Where did you get ChordPro from? GitHub
 - ChordPro version: 5.988_020
 - Perl version: v5.34.0

**Comment by nomike** [comment]
I just figured out that if you put this in the config, it works:

```plaintext
    "pdf": {
        "kbdiagrams" : {
            "show"     :  "right"
        }
    }
```

But the question still remains as of why keyboard diagrams are not shown in the same location by default as diagrams for string instruments.

**Comment by sciurius** [comment]
This is what 5.988_20 produces here, with your original files and the command
````
chordpro --nodefaultconfig --config keyboard.json test.cho --out=test.pdf
````

[test.pdf](https://github.com/ChordPro/chordpro/files/9276939/test.pdf)

**Comment by sciurius** [comment]
Where did you find this documented?

> I'm also questioning the decision to show guitar chords in the right column by default and keyboard diagram on the bottom of the first page. Even as it's documented, why treat them differently?

**Comment by nomike** [comment]
https://www.chordpro.org/chordpro/chordpro-configuration-pdf/#keyboard-diagrams

> By default the diagrams are at the end of the song but it is also possible to have them at the bottom, or in a side column on the first page of the song. See [Chords diagrams in a side column](https://www.chordpro.org/chordpro/chordpro-configuration-pdf/#chords-diagrams-in-a-side-column) below.

**Comment by sciurius** [comment]
Fixed in 3cd40ff6502b0a9e92397f36ec919f55f6e51de0
